fl4health/clients/ditto_client.py

```diff
@@ -112,7 +119,10 @@ def get_parameters(self, config: Config) -> NDArrays:
             NDArrays: GLOBAL model weights to be sent to the server for aggregation
         """
         if not self.initialized:
-            log(INFO, "Setting up client and providing full model parameters to the server for initialization")
```

Comment:
Not a big deal at all but it looks like a lot of formatting related changes crept into the PR, may want to check the settings of your formatter

Reply:
Yeah my bad I prefer to have my formatter set at 90 characters instead of 120 but I'll turn off the autoformat on save setting so that I don't mess with other peoples files.